dbcmetrics/dbcm_logic/dbcstorage.py

```diff
@@ -0,0 +1,177 @@
+'''
+Module for a class that queries a s3 bucket for their size and the number of files to provide them as Prometheus Gauge Metrics
+'''
+import logging
+import os
+from re import I
+from time import sleep
+import threading, requests, json, urllib.parse
+from typing import Dict
+from typing import List
+from prometheus_client import Gauge, Info
+import boto3
+from botocore.config import Config
+
+
+class StorageMetricsThreading(object):
+    '''
+    Module to get metrics about an object storage
+    '''
+    def __init__(self):
+        self.storage_interval = int(os.getenv("STORAGE_INTERVAL"))
+        self.storage_exclude_subfolders = os.getenv("STORAGE_EXCLUDE_SUBFOLDERS").lower()
+        self.storage_provider_url = os.getenv("STORAGE_PROVIDER_URL")
+        self.storage_provider_region = os.getenv("STORAGE_PROVIDER_REGION")
+        self.bucket_name = os.getenv("BUCKET_NAME")
```

Comment:
I would suggest to use a STORAGE_ prefix for BUCKET_NAME, ACCESS_KEY and ACCESS_SECRET

Reply:
Yeah, that bugs me a bit because the current naming comes from the field name of the [s3 Item](https://start.1password.com/open/i?a=RZRVBZLAI5D4LEX2LEZGQPX5X4&v=k5fi3siathzvnqsucgu7e4uycu&i=p5dtut3ikpfbzynloxvnivlyuy&h=hpi-schul-cloud.1password.eu) in the 1Password Vaults, so the secret env variables are either inconsistent with the other env variables for configs or with the secrets in 1Password.
I would prefer to keep the consistence at least within the application, but, as far as I know, there is no way of changing the name of the key while importing them with a OnePasswordItem CRD as a k8s secret. So all I could do would be to rename the vars within the class to
```suggestion
        self.storage_bucket_name = os.getenv("BUCKET_NAME")
```
and the likes.

Reply:
Class variable and environment variable are now renamed because the latest commit in the according [infra-schulcloud PR-139](https://github.com/hpi-schul-cloud/infra-schulcloud/pull/139) changes the environment variable creation from secrets via the helm chart by adding `STORAGE_´ as a prefix.